modules/yieldlabBidAdapter.js

```diff
@@ -159,6 +166,16 @@ function isVideo (format, adtype) {
   return utils.deepAccess(format, 'mediaTypes.video') && adtype.toLowerCase() === 'video'
 }
 
+/**
+ * Is this a native format?
+ * @param {Object} format
+ * @param {String} adtype
+ * @returns {Boolean}
+ */
+function isNative(format, adtype) {
+  return utils.deepAccess(format, 'mediaTypes.native') && adtype === 'NATIVE'
```

Comment:
Maybe check the ad type like in `isVideo`? The check would if it would be `nAtIvE`. Not sure if this could ever happen.

Reply:
Agree, better to align with the rest code